runtime/runtime.go

```diff
@@ -476,3 +477,47 @@ func For(ctx *Context, args ...parser.Value) parser.Value {
 	}
 	return parser.NewAny(nil, nil)
 }
+
+//Dump the context content
+func (c *Context) Dump() {
+	for id, val := range c.env {
+		fmt.Println(id, val)
```

Comment:
print?

Reply:
I am using that func as debug tool. I accept suggestions.

Reply:
maybe something in rumtime or rumtime/debug can help with this

Reply:
We need to start thinking about package, the dump would enter the `debug` (#95) package.
more issue: #96